src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -128,13 +128,107 @@ public class Converter implements Callable<Void> {
   /** NGFF specification version.*/
   public static final String NGFF_VERSION = "0.4";
 
+  private volatile Path inputPath;
+  private volatile String outputLocation;
```

Comment:
Any reason to keep `outputLocation` or is it also worth unifying the internal variable name with the getter/setter?

Reply:
`outputPath` is defined later on, and is resolved from `outputLocation` as either a path on disk or an s3 location. I can rename `outputLocation` to `outputPath` if you feel that's clearer, but then would need to rename the current `outputPath` to something else.